scrapy_poet/middleware.py

```diff
@@ -72,11 +73,12 @@ def process_request(self, request: Request, spider: Spider) -> Optional[DummyRes
             return None
 
         logger.debug(f"Using DummyResponse instead of downloading {request}")
+        self.crawler.stats.inc_value("downloader/request_count/skipped")
```

Comment:
Is it a general Scrapy issue, that if a response is returned in process_request, a request is not counted? If so, would it make more sense to have a stat like this in Scrapy itself? I'm worried that we're reusing Scrapy's builtin namespace for stats here.

Reply:
In Scrapy these stats are set from https://github.com/scrapy/scrapy/blob/4bdaa54af4470582845206a9fa21779f80a4b123/scrapy/downloadermiddlewares/stats.py#L22-L50

So I see no easy way to address this upstream. I am not even sure that it needs addressing, as [I mentioned](https://github.com/scrapinghub/scrapy-poet/pull/76#discussion_r897961022).

However, if the issue is the builtin namespace, and that makes sense, shall we simply switch the namespace to something like `scrapy_poet/skipped_request_count` or `scrapy_poet/request_count/skipped`?

Reply:
I think that's fine to use another namespace; I'm not sure about the use case, but and extra stat value won't hurt. 

Something like `scrapy_poet/dummy_response_count`? `scrapy_poet/request_count/skipped` is also ok, but it reads as scrapy-poet skips some requests, while we're increasing it where scrapy-poet is actually handling this request. 

If naming is an issue, we may also skip the stats increase for now, and handle it separately. But anyways, I'm fine with merging it, as soon as Scrapy's namespace is not used :)